config/maintainers.json

```diff
@@ -10,6 +10,36 @@
       "link_url": null,
       "name": null,
       "show_on_website": false
+    },
+    {
+      "alumnus": false,
+      "avatar_url": "https://avatars0.githubusercontent.com/u/7527155?v=4&s=460",
+      "bio": null,
+      "github_username": "Stargator",
+      "link_text": null,
+      "link_url": null,
+      "name": null,
+      "show_on_website": false
```

Comment:
Consider showing yourself on the website. You'd end up on the v2 site (not the v1) which is not live yet. Here's the page: https://v2.exercism.io/team

Reply:
I'm guessing the website pulls bio from the github profile. If so, why is there a bio field?

Reply:
Yepp, it does pull the bio if the bio is empty, but often people want to have a different bio depending on the language track, that says more about how they got into the language or what kind of work they focus on for the track.